src/bindings/python/flux/asyncio/job.py

```diff
@@ -0,0 +1,75 @@
+"""Flux-asyncio integration for `flux.job` functions."""
+
+import asyncio
+
+import flux.job
+from flux.asyncio import _get_loop_flux_handle
+
+
+async def submit(*args, **kwargs):
```

Comment:
At the risk of making this control flow more complicated with another level of indirection, I wonder if we can avoid some of the repetition with either a function call or a decorator.  Just as a rough strawman of what I'm thinking of:
```
async def _integrate_with_async(async_func, *args, sync_func=None, **kwargs):
    handle = _get_loop_flux_handle()
    fut = asyncio.get_running_loop().create_future()
    if sync_func is None:
    	getter_func = lambda flux_fut: fut.set_result(flux.job.submit_get_id(flux_fut))
    else:
    	getter_func = lambda flux_fut: fut.set_result(flux_fut.get())
    async_func(handle, *args, **kwargs).then(getter_func)
    return await fut
    
async def submit(*args, **kwargs):
	return await _integrate_with_async(flux.job.submit_async, flux.job.submit_get_id)
	
async def cancel(*args, **kwargs):
	return await _integrate_with_async(flux.job.cancel_async)
```

Reply:
That's a great idea, I will add that in.